src/index.html

```diff
@@ -591,6 +591,66 @@ <h4>React Remixed</h4>
                 </div>
               </article>
             </li>
+            <li>
+              <article class="talks-box">
+                <img
+                  class="talks-box__image"
+                  src="images/speakers/2022/speaker-przemyslaw-beigert.png"
+                  alt="Photo of speaker: Przemysław Beigert"
+                />
+                <div class="talks-box__info">
+                  <div class="talks-box__header">
+                    <div>
+                      <h3>Przemysław Beigert</h3>
+                      <a
+                        href="https://twitter.com/przemyslawjanp"
+                        target="_blank"
+                        title="Twitter profile - Przemysław Beigert"
+                        >#przemyslawjanp</a
+                      >
+                    </div>
+                  </div>
+
+                  <h4>Advanced TypeScript</h4>
+                  <p class="talks-box__desc">
+                    TypeScript become extremely popular tool but how well you know him? Probably you
```

Comment:
This is not another introduction to the Vue. I'm gonna focus on differences between Vue, React and Angular in topics like: e.g.: communication, forms, storage, testing. All with code examples. Base on that I'll tell when Vue will be good choose and why invest time in learning it isn't good idea.